packages/contact-exchange/package.json

```diff
@@ -20,5 +20,11 @@
   "keywords": [],
   "author": "adorsys",
   "license": "ISC",
-  "description": ""
+  "description": "",
+  "dependencies": {
+    "@adorsys-gis/multiple-did-identities": "file:../packages",
+    "axios": "^1.7.7",
+    "did-resolver-lib": "https://gitpkg.vercel.app/adorsys/didcomm-messaging-clients-utilities/libs/did-resolver-lib?main",
+    "didcomm-node": "^0.4.1"
+  }
```

Comment:
Please check that this dependency works in the browser.

Reply:
It does works